types/connection.d.ts

```diff
@@ -121,6 +121,12 @@ declare module 'mongoose' {
      */
     readonly id: number;
 
+    /**
+     * Helper for MongoDB Node driver's `listCollections()`.
+     * Returns an array of collection names.
+     */
+    listCollections(collection: string): Promise<Pick<mongodb.CollectionInfo, 'name' | 'type'>[]>;
```

Comment:
I'm a bit confused, why does `listCollections` take a `collection` parameter? I can see that the implementation in `lib/connection.js` is not defining or passing any parameters to the driver's `listCollections` method.

Reply:
Also, maybe a TS test for this would be nice.

Reply:
```suggestion
    listCollections(): Promise<Pick<mongodb.CollectionInfo, 'name' | 'type'>[]>;
```

the jsdoc does not mention any parameter and the js also does not have any parameter, so i also think this is not correct
(likely also a copy-paste from `dropCollection`?)